tests/scripts/pgocheck/pefile/ordlookup/__init__.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+from . import ws2_32
+from . import oleaut32
+
+'''
+A small module for keeping a database of ordinal to symbol
+mappings for DLLs which frequently get linked without symbolic
+infoz.
+'''
+
+ords = {
+    b'ws2_32.dll': ws2_32.ord_names,
```

Comment:
Can you tell me what this is for?  I see a lot of stuff around Windows specific DLLs (ws2_32, oleaut32, etc.) but I'm not sure how it relates to validating that the DLL was compiled with PGO.